src/javascript/charlesetienneV-consoleLogAPrompt.js

```diff
@@ -1,5 +1,5 @@
 let x = prompt('Enter a word then check console: ');
 
 function consoleLogTheXPrompt(x) {
```

Comment:
@Charlesetiennev rename the function to be same as in the filename, or vice versa